pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -4,67 +4,10 @@
 import numpy as np
```

Comment:
I recommend changing the module doc string to: "Test basic functionality for loading Earth relief datasets."